actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -53,6 +59,8 @@ class ActionCable.Connection
     @isState("open", "connecting")
 
   # Private
```

Comment:
Sorry to nitpick, but add a newline after this comment please. Indicates that all methods below are "private", not just the `isPorotocolSupported` method.